test/e2e/tests/test_replicationgroup.py

```diff
@@ -122,6 +131,29 @@ def rg_cmd_fromsnapshot(bootstrap_resources, make_rg_name, make_replication_grou
     rg_deletion_waiter.wait(ReplicationGroupId=input_dict["RG_ID"])
 
 
+@pytest.fixture(scope="module")
+def rg_cmd_update_input(make_rg_name):
+    return {
+        "RG_ID": make_rg_name("rg-cmd-update"),
+        "ENGINE_VERSION": "5.0.0",
+        "NUM_NODE_GROUPS": "1",
+        "REPLICAS_PER_NODE_GROUP": "1"
+    }
+
+
+@pytest.fixture(scope="module")
+def rg_cmd_update(rg_cmd_update_input, make_replication_group, rg_deletion_waiter):
+    input_dict = rg_cmd_update_input
```

Comment:
What's the purpose of this temporary variable?

Reply:
I guess it just makes the below lines slightly more readable, but it's not at all necessary